go/interpreter/lfvm/interpreter.go

```diff
@@ -659,18 +649,3 @@ func _precomputeStackLimits() [256]stackLimits {
 	}
 	return res
 }
-
-func isWriteInstruction(opCode OpCode) bool {
```

Comment:
is there already a test that checks that these opcodes produce `ERROR` status ?

Reply:
yap: 
https://github.com/Fantom-foundation/Tosca/blob/adfa2779d6536498ed286d3f25c0f87724e52a43/go/interpreter/lfvm/interpreter_test.go#L680